pkg/local/server/commands.go

```diff
@@ -231,30 +231,13 @@ func CloneLV(ctx context.Context, src, dest string) (string, error) {
 }
 
 // ListVG get vg info
-func ListVG() ([]*lib.VG, error) {
-	args := []string{NsenterCmd, "vgs", "--units=b", "--separator=\"<:SEP:>\"", "--nosuffix", "--noheadings",
-		"-o", "vg_name,vg_size,vg_free,vg_uuid,vg_tags,pv_count", "--nameprefixes", "-a"}
-	cmd := strings.Join(args, " ")
-	out, err := utils.Run(cmd)
+func ListVG() ([]lib.VG, error) {
+	out, err := utils.CommandOnNode("vgs", "--reportformat", "json", "--units", "b", "-o", "vg_name,vg_size,vg_free,vg_uuid,vg_tags,pv_count", "--nosuffix").Output()
```

Comment:
what's the differences without `--noheading`, `--nameprefixes`, `-a`?

Reply:
`--noheading` hides the heading in basic report, it is not relevant in json report format.
`--nameprefixes` produces field=value output not relevant in json report format.
`-a` outputs all VGs, since we are not specifying any VG here, this is the default.